tests/unittest_brain_qt.py

```diff
@@ -93,7 +93,7 @@ def test_slot_disconnect_no_args() -> None:
     reason="These tests require the PyQt5, PySide2, PyQt6, and PySide6 libraries.",
 )
 @pytest.mark.skipif(
-    PY311_PLUS, reason="PySide2>=5.15.2 not yet available for Python 3.11"
+    PY311_PLUS, reason="PySide2>=5.15.2 and PySide6 not yet available for Python 3.11"
 )
```

Comment:
I think this is unnecessary as we already have line 88 a,d 90 to handle that.

Reply:
Ah, I see. Instead I should have simply put the restriction in `requirements_test_brain.txt`:

```
PyQt5>=5.15.2
PySide2>=5.15.2; python_version<"3.11"
PyQt6
PySide6; python_version<"3.11"
```

I'll remove this skip line.